gulpfile.js

```diff
@@ -1,36 +1,31 @@
-'use strict'
```

Comment:
Just out of interest, why are we removing 'use strict'?

Reply:
I think eslint has this rule because it is assuming everyone is using 'babel' that adds it for you. Do we know if uglifier does this too? If not it might be safer to disable this rule as it does change the browser's behaviour.